examples/custom_checkers/r9901_impossible_condition.py

```diff
@@ -0,0 +1,28 @@
+def print_natural_number(x: int):
+    """
+    Print number x only if it's a natural number.
```

Comment:
See my above comment about descriptions.

But also pick a different example, don't introduce the terminology of natural numbers here (this is unnecessarily complex for these examples)